esp/bindings/http/platform/httptransport.ipp

```diff
@@ -151,6 +152,7 @@ public:
     const char *queryHost(){return m_host.get();}
     virtual void setPort(int port) {m_port = port;};
     virtual int getPort() {return m_port;};
+    virtual bool getHasPortInHost() const {return hasPortInHost;};
```

Comment:
minor: extra ; after the function definition (same as setPort and other functions above).
Also, this doesn't need to be virtual - it isn't in a base class or overridden.

Reply:
Too many old places with the extra ; and virtual. I will only clean them from line 153, 154, and 155.